lib/matplotlib/tests/test_axes.py

```diff
@@ -2108,68 +2108,59 @@ def layers(n, m):
     axs[1, 1].stackplot(range(100), d.T, baseline='weighted_wiggle')
 
 
+def _bxp_test_helper(
+        stats_kwargs={}, transform_stats=lambda s: s, bxp_kwargs={}):
```

Comment:
this isn't a big deal. feel free to ignore it. but my early morning, uncaffeinated brain has a hard time with function definitions whose first parameters is on a second line